test_scripts/API/AddCommand/SDL_resends_WARNINGS_to_MOB_when_HMI_respond_WARNINGS_to_one_HMI_portions.txt

```diff
@@ -0,0 +1,14 @@
+./test_scripts/API/SetGlobalProperties/ATF_SetGlobalProperties_TTS_SetGlobalProperties_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_TTS_SetGlobalProperties
```

Comment:
@vvvakulenko, In the CRQ is listed in Information section:
1. RPC CreateInteractionChoiceSet.
2. DeleteInteractionChoiceSet
Please clarify are these two RPCs should be also tested in scope of this CRQ.

3. "List of **_more_** impacted RPCS"
Because of "more", please clarify is there any other RPCs that should be tested in scope of this CRQ.